app/Nova/Resources/Integration.php

```diff
@@ -150,21 +152,29 @@ public function actions(NovaRequest $request): array
                 ->confirmText('Are you sure you want to activate this integration?')
                 ->confirmButtonText('Activate')
                 ->cancelButtonText("Don't activate")
-                ->canSee(fn () => $this->canBeActivated())
+                ->canSee(fn (Request $request) => $request instanceof ActionRequest || $this->canBeActivated())
                 ->canRun(fn (Request $request, IntegrationModel $model) => $model->canBeActivated()),
 
+            (new ApproveIntegration(App::make(IntegrationRepository::class)))
+                ->exceptOnIndex()
+                ->confirmText('Are you sure you want to approve this integration?')
+                ->confirmButtonText('Approve')
+                ->cancelButtonText("Don't approve")
```

Comment:
Would maybe put `Reject` to avoid having the positive action work on the danger action

Reply:
I think it is a `Cancel` because a `Reject` does not exist.

Reply:
I changed everything to `Cancel` except for the `Don't distribute` because those will be removed in another pull request.